src/registry_schemas/schemas/agm_extension.json

```diff
@@ -12,16 +12,66 @@
             "type": "object",
             "required": [
                 "year",
-                "isFirstAgm"
+                "isFirstAgm",
+                "prevAgmRefDate",
+                "extReqForAgmYear",
+                "expireDateCurrExt",
+                "intendedAgmDate",
+                "totalApprovedExt",
+                "expireDateApprovedExt"
             ],
             "properties": {
                 "year": {
-                    "type": "string",
+                    "type": "integer",
+                    "minimum": 1000,
+                    "maximum": 9999,

```

Comment:
This is now inconsistent with the year definition in the AGM Location schema. It would be best if they were both the same. (Maybe the other one should be updated.)

What does the Legal API / db expect this field to be typed as? Do we need to do any math with this field (eg, current_year - year <= 2)?

Reply:
Obsolete conversation. Resolving.